robot_smach_states/src/robot_smach_states/manipulation/point_at.py

```diff
@@ -0,0 +1,103 @@
+# ROS
+import rospy
+import smach
+
+# TU/e Robotics
+from robot_skills.arms import PublicArm
+from robot_skills.util.entity import Entity
+import robot_smach_states.util.designators as ds
+
+# System
+import numpy as np
+import sys
+
+
+class PointAt(smach.State):
+    def __init__(self, robot, arm_designator, point_at_designator, look_at_designator=None):
+        """
+        Drive the robot back a little and move the designated arm to place the designated item at the designated pose
+        :param robot: Robot to execute state with
+        :param placement_pose: Designator that resolves to the pose to place at. E.g. an EmptySpotDesignator
+        :param arm: Designator -> arm to place with, so Arm that holds entity_to_place, e.g. via
+        ArmHoldingEntityDesignator
+        """
+        smach.State.__init__(self, outcomes=['succeeded', 'failed'])
+
+        if look_at_designator is None:
+            look_at_designator = point_at_designator
```

Comment:
Why not let look_at_designator be None and resolve this in the execute()?

Reply:
Because of the type-checks a bit below.